bin/run-addon

```diff
@@ -115,7 +115,7 @@ if [[ -z "$no_auto" ]] ; then
 fi
 
 cd build/addon
-jpm run --binary $binary --profile ../../Profile --no-copy \
+jpm run --binary $binary --profile ../../dist/Profile --no-copy \
```

Comment:
Is the dist directory in my local checkout vestigial then? Where is the correct location of the new profile?